lib/drag_time_api_web/router.ex

```diff
@@ -2,6 +2,7 @@ defmodule DragTimeApiWeb.Router do
   use DragTimeApiWeb, :router
 
   pipeline :api do
+    plug CORSPlug, origin: "*"
```

Comment:
Guessing this solves our problem we brought up a couple hours ago?

Reply:
It looks like the asterisk allows anybody to access our API

Reply:
Yeah! It allows for other domains outside of localhost:4000 to get data